File: solutions/0476-number-complement/solution.py
class Solution(object):
    def findComplement(self, num):
        """
        :type num: int
        :rtype: int
        """
        b=""
        for i in bin(num)[2:]:
            if i=='0':
                b+='1'
            else:
                b+='0'
        return int(b,2)

File: solutions/0476-number-complement/solution.java
class Solution {
    public int findComplement(int num) {
        String b="";
        for(char i: Integer.toBinaryString(num).toCharArray())
            if (i=='0')
                b+='1';
            else
                b+='0';
        return Integer.parseInt(b,2);
    }
}
